src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -151,11 +158,18 @@ private static void usage() {
         + "   to start a server, you can optionally password protect the game using triplea.server.password=foo");
   }
 
-  public static void main(final String[] args) {
+  public static void main(String[] args) {
+    if( args == null ) {
+      args = new String[0];
+    }
+    if( exceptionHandler == null ) {
+      exceptionHandler = new ErrorHandler();
```

Comment:
In the case where a test injects the exception handler, we would be creating an ErrorHandler object needlessly. Not a big cost, and we get to get rid of this if check, I agree and will make the change.